rpc_jobs/rpc_leapfrog.yml

```diff
@@ -0,0 +1,194 @@
+              Prepare Deployment
+              Deploy RPC w/ Script
+              Setup MaaS
+              Verify MaaS
+              Install Tempest
+              Tempest Tests
+              Leapfrog
+              Prepare Kibana Selenium
+              Kibana Tests
+              Holland (test holland mysql backup)
+              Upgrade
+              Pause (use to hold instance for investigation before cleanup)
+              Cleanup
+              Destroy Slave
+    triggers:
+      - timed: "{CRON}"
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_cit_all.*|.*recheck_cit_{context}.*'
+          only-trigger-phrase: false
+          white-list-target-branches:
+            - "{branches}"
+          auth-id: "github_account_rpc_jenkins_svc"
+          status-context: 'CIT/{context}'
+          cancel-builds-on-update: true
+
+    dsl: |
+      // CIT Slave node
+      currentBuild.result = 'SUCCESS'
+      timeout(time: 10, unit: 'HOURS'){{
+        node() {{
+          try {{
+            dir("rpc-gating") {{
+              git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+              common = load 'pipeline_steps/common.groovy'
+              pubcloud = load 'pipeline_steps/pubcloud.groovy'
+              aio_prepare = load 'pipeline_steps/aio_prepare.groovy'
+              leapfrog = load 'pipeline_steps/leapfrog.groovy'
+              deploy = load 'pipeline_steps/deploy.groovy'
+              tempest = load 'pipeline_steps/tempest.groovy'
+              holland = load 'pipeline_steps/holland.groovy'
+              maas = load 'pipeline_steps/maas.groovy'
+              kibana = load 'pipeline_steps/kibana.groovy'
+            }}
+            // We need to checkout the rpc-openstack repo on the CIT Slave
+            // so that we can cater for builds on the more restricted images
+            // used for artifact-based builds.
+            if (env.STAGES.contains("Upgrade")) {{
+              common.prepareRpcGit(env.UPGRADE_FROM_REF, env.WORKSPACE)
+            }} else {{
+              common.prepareRpcGit("auto", env.WORKSPACE)
+            }} // if
+            if(common.is_doc_update_pr("${{env.WORKSPACE}}/rpc-openstack")){{
+              return
+            }}
+            pubcloud.runonpubcloud {{
+              // try within pubcloud node so we can archive archive_artifacts
+              // after a failure, before the node is cleaned up.
+              try {{
+                environment_vars = [
+                  "DEPLOY_HAPROXY=yes",
+                  "DEPLOY_AIO=no",
+                  "DEPLOY_MAAS=no",
+                  "DEPLOY_TEMPEST=no",
+                  "DEPLOY_SWIFT=${{DEPLOY_SWIFT}}",
+                  "DEPLOY_CEPH=${{DEPLOY_CEPH}}",
+                  "DEPLOY_ELK=${{DEPLOY_ELK}}",
+                  ]
+                # TODO(evrardjp) add other leapfrog vars to alter behavior
```

Comment:
Hey @evrardjp, I'll review this more closely once it's passing the lint job -- see http://groovy-lang.org/syntax.html#_single_line_comment as `#` is not valid for comments.